visualization/app/codeCharta/ui/actionIcon/actionIcon.component.scss

```
@@ -0,0 +1,23 @@
+@use "../../../material/theme";
+
+cc-action-icon {
+	display: inline-flex;
+	align-items: center;
+	cursor: pointer;
+	width: 24px;
+	height: 24px;
+	border-radius: 100%;
+	background-color: theme.$cc-primary-color;
+	font-size: 16px;
```

```suggestion
	font-size: 14px;
```
looks a bit better when we leave some space

---

done with next push